fec.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
   hotReload: process.env.HOT === 'true',
   routes: {
     ...(process.env.CONFIG_PORT && {
-      '/api/chrome-service/v1/static': {
+      '/api/chrome-service/v1/': {
```

Comment:
Unless you want to run the entire backend for configs, you should keep the /static in the pathname